guardian/VaultDoor
# Automatic Login Refreshing

## What does this change?
Adds code which attempts to refresh the login data two minutes before it expires. The code is heavily based on existing code from pluto-headers.

## How to test
The software should refresh the login data two minutes before it expires. You should see 'Login refreshed' in the console after a successful refresh.

## How can we measure success?
The login data gets refreshed.

This was tested on a machine running macOS 10.12.6 and on three virtual machines.

@fredex42
